resources/cmake/ConnextDdsAddSecurityArtifacts.cmake

```diff
@@ -0,0 +1,186 @@
+# (c) 2021 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+# No duplications, whole or partial, manual or electronic, may be made
+# without express written permission.  Any such copies, or revisions thereof,
+# must display this notice unaltered.
+# This code contains trade secrets of Real-Time Innovations, Inc.
+
+#[[.rst:
+.. _connextdds_add_security_artifacts:
+
+ConnextDdsAddSecurityArtifacts
+------------------------------
+
+Generate the artifacts needed for secure examples.
+
+``connextdds_add_security_artifacts()``
```

Comment:
`add` is not a really good name because it is not clear if they are generated and they will copied or what.

Replace the name of the macro by `connextdds_generate_security_artifacts`